src/components/ReactPlayer/ReactPlayer.tsx

```diff
@@ -89,7 +89,7 @@ export const ReactPlayerBlock = React.forwardRef<ReactPlayerBlockHandler, ReactP
             ariaLabel,
             ratio,
             autoRatio,
-            contain,
+            contain = true,
```

Comment:
This looks like a non-backward compatible change.

Reply:
Why is this change necessary?